### PR TITLE
freeswitch 1.10.7

### PR DESCRIPTION
**Pull request by EricFromCanada**
Created with `brew bump-formula-pr`.

`resource` blocks may require updates.

### CODE REVIEW

**Comment by EricFromCanada** [comment]
```
./vpx_ports/x86.h:313:3: error: invalid output constraint '=a' in asm
  cpuid(0, 0, reg_eax, reg_ebx, reg_ecx, reg_edx);
  ^
./vpx_ports/x86.h:49:26: note: expanded from macro 'cpuid'
                       : "=a"(ax), "=b"(bx), "=c"(cx), "=d"(dx) \
```
Failing on ARM due to bundled libvpx not detecting the platform correctly. https://github.com/signalwire/freeswitch/issues/1450

**Comment by carlocab** [comment]
Can we get this to use our `libvpx`?

File: Formula/freeswitch.rb [hunk 1]
```diff
@@ -186,12 +155,15 @@ def install
...
-                          "--enable-static",
-                          "--prefix=#{prefix}",
-                          "--exec_prefix=#{prefix}"
+    args = std_configure_args + %W[
+      --enable-shared
+      --enable-static
+      --exec_prefix=#{prefix}
+    ]
+    # Fails on ARM: https://github.com/signalwire/freeswitch/issues/1450
+    args << "--disable-libvpx"
```
Comment (carlocab) [inline Formula/freeswitch.rb:164]:
```suggestion
    args << "--disable-libvpx" if Hardware::CPU.arm?
```
?

Comment (EricFromCanada) [reply Formula/freeswitch.rb:164]: Wasn't sure if having inconsistent codec support between architectures would be a problem, so just disabled it for all.

Comment (carlocab) [reply Formula/freeswitch.rb:164]: We usually just disable things on ARM if they don't work specifically on ARM though.

**Comment by carlocab** [comment]
I think

https://github.com/Homebrew/homebrew-core/blob/5d90c0d5ab3c128a007516fde6a6a2dc33ad5a14/Formula/freeswitch.rb#L156-L158

has been fixed by https://github.com/signalwire/freeswitch/commit/c985c90cbb93da152fb7cd260cf6ff356d5244d1.

**Comment by carlocab** [comment]
Is the bundled `libvpx` new? I'm just wondering why it didn't fail before. Or is this a `libvpx` regression?

**Comment by EricFromCanada** [comment]
@carlocab Up until [this commit](https://github.com/signalwire/freeswitch/commit/fb14eef2b4214dd9aacf1111e28ad8633951bdd4), libvpx was simply omitted from the build on macOS 11.x and up.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**